packages/endpoints-overlay/src/endpoint.js

```diff
@@ -18,7 +18,31 @@ import {
 
 import * as Styled from "./styled";
 
-function Icon({ type }) {
+function DefaultMapMarkerIcon({ location, type }) {
+  return (
+    <Styled.StackedIconContainer title={location.name}>
+      {type === "from" ? (
+        // From icon should have white circle background
+        <>
+          <Styled.StackedCircle size={24} />
+          <Styled.StackedLocationIcon size={24} type={type} />
+        </>
+      ) : (
+        <>
+          <Styled.StackedToIcon size={24} type="to" />
+          <Styled.ToIcon size={20} type={type} />
+        </>
+      )}
+    </Styled.StackedIconContainer>
+  );
+}
+
+DefaultMapMarkerIcon.propTypes = {
+  location: locationType.isRequired,
+  type: PropTypes.string.isRequired
+};
+
+function UserLocationIcon({ type }) {
```

Comment:
This could use jsdoc. I had trouble remembering what these icons were for at first.